test_framework/terraform/aws/centos/main.tf

```diff
@@ -268,4 +284,95 @@ resource "aws_ebs_volume" "lh_aws_hdd_volume" {
   tags = {
     Name = "lh-aws-hdd-volume-${random_string.random_suffix.id}-${count.index}"
   }
+}
+
+# Create load balancer for rancher
+resource "aws_lb_target_group" "lh_aws_lb_tg_443" {
```

Comment:
For each new resource, need to add a tag for our usage identity.

Reply:
Modified.